xmtp/src/storage/encrypted_store/mod.rs

```diff
@@ -224,10 +224,10 @@ impl EncryptedMessageStore {
         conn: &mut PooledConnection<ConnectionManager<SqliteConnection>>,
         user_address: &str,
         timestamp: i64,
-    ) -> Result<usize, StorageError> {
-        diesel::update(users::table.filter(users::user_address.eq(user_address)))
+    ) -> Result<StoredUser, StorageError> {
+        diesel::update(users::table.find(user_address))
             .set(users::last_refreshed.eq(timestamp))
-            .execute(conn)
+            .get_result::<StoredUser>(conn)
```

Comment:
This change just ensures that an error is thrown if no row exists and no update was made (`get_result` requires there to be one result).